packages/starlight/__e2e__/fixtures/ssr/src/content/docs/404.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Home Page
+template: splash
+lastUpdate: true
+hero:
+  title: Make your docs shine with Starlight
+  tagline: Everything you need to build a stellar documentation website. Fast, accessible, and easy-to-use.
+  actions:
+    - text: Get started
+      icon: right-arrow
+      variant: primary
+      link: /getting-started/
+    - text: View on GitHub
+      icon: external
+      link: https://github.com/withastro/starlight
```

Comment:
I think we are only testing the `#server-check` element on a 404 page and not anything related to the hero so maybe we could remove this entirely to simplify the fixture page?